docs/source/api/plugin/landing-pages.md

```diff
@@ -106,6 +106,52 @@ By default, the landing page displays a footer that links to the documentation t
 <tr>
 <td>
 
+###### `document`
```

Comment:
do these get passed through to sandbox in the regular landing page? We were just missing them before but they are supported right?